### ISSUE TITLE
Error loading Latex_env

### ISSUE THREAD

**Issue by dmyersturnbull** [ipython-contrib/jupyter_contrib_nbextensions#643]
I followed the installation instructions at https://github.com/ipython-contrib/IPython-notebook-extensions/blob/master/README.md, downloaded https://github.com/ipython-contrib/IPython-notebook-extensions/blob/master/nbextensions/usability/latex_envs/doc/html/latex_env_doc.ipynb, and ran it in this Anaconda environment:
```
jupyter                   1.0.0                    py35_3  
jupyter_client            4.2.2                    py35_0  
jupyter_console           4.1.1                    py35_0  
jupyter_core              4.1.0                    py35_0  
```

Here's the error log when I run `%%javascript 
IPython.utils.load_extensions('calico-document-tools');`:
```
main.min.js:12156 Failed to load extension: ["nbextensions/calico-document-tools"] Error: Script error for "nbextensions/calico-document-tools"
http://requirejs.org/docs/errors.html#scripterror
    at makeError (http://localhost:8888/static/components/requirejs/require.js?v=6da8be361b9ee26c5e721e76c6d4afce:165:17)
    at HTMLScriptElement.onScriptError (http://localhost:8888/static/components/requirejs/require.js?v=6da8be361b9ee26c5e721e76c6d4afce:1732:36)(anonymous function) @ main.min.js:12156
```
LaTeX is not rendered in subsequent cells. Am I supposed to need calico-document-tools? If so, that's not clear from the documentation.

**Comment by jfbercher** [comment]
The icalico extension is something else that was used for convenience in the docs. You do not need it and it has nothing to do with the latex_envs extension. 
The installation instructions in latex_envs doc file are a bit outdated (regarding paths in particular). You are welcome to update it. The best is to use the global installation procedure for all extensions, see readme, and then enable latex_envs in the nbextension tab.

**Comment by dmyersturnbull** [comment]
Okay, good to know. However, LaTeX isn't rendered, and nothing displays in the console.

> The best is to use the global installation procedure for all extensions, see readme, and then enable latex_envs in the nbextension tab.

That's what I did:
`pip install https://github.com/ipython-contrib/IPython-notebook-extensions/archive/master.zip --user`
And then enabled latex_envs in the nbextension tab.

**Comment by jfbercher** [comment]
I can confirm this from a fresh install. 

@jcb91 
- The problem comes from commit https://github.com/ipython-contrib/IPython-notebook-extensions/commit/30334d2c54a7aadbc076357dc511a066ca00e376#diff-cc4054765e888c4b375cd13fed136291 where we wanted to use relative paths. This does not work  and fails silently (since the js files are required but not found, actually the system waits for them forever). I was confident and unfortunately did not test. [And I am always late in refreshiong my install where I have some "devs" I want to preserve]
- I think that we should also remove https://github.com/ipython-contrib/IPython-notebook-extensions/commit/37c3865a1ef93ce29a8687f6533629726159cad0#diff-cc4054765e888c4b375cd13fed136291 which introduces a simplified require wrapper. Indeed, I am afraid that because js loading is asynchronous, initialization eg in ` var maps = initmap();` car be called before `var initN = require('./initNb');` is actually loaded. 

If possible, the most simple would be to revert these two commits. And/or find why the relative paths do not work. Otherwise we should make a corrective commit to revert both.

**Comment by jcb91** [comment]
Oh dear :( apologies!

I'm away for the weekend so can't test till I get back. My understanding of
the simplified require is that it ensures that a `require('blah')` call
inside the simplified wrapper will execute synchronously (since require
parses the wrapper for such calls and resolves them before executing the
wrapper). I guess I got the relative URL wrong?

**Comment by jfbercher** [comment]
@jcb91 yes the wrapper was right but unfortunately not the relative paths. Also I removed a function wrapper which turned functions in that file into local functions. 

- I tried to address the issue in #645

@dmyersturnbull just edit `latex_envs.js` and `initNb.js` as in https://github.com/ipython-contrib/IPython-notebook-extensions/pull/645/files#diff-cc4054765e888c4b375cd13fed136291  and it should work. 

There is still lot of to do and improve on the latex_envs project. Perhaps in august.
